build.sbt

```diff
@@ -37,7 +37,7 @@ lazy val publishSettings = Seq(
 
 lazy val commonSettings = Seq(
   scalacOptions ++= Seq("-deprecation", "-feature", "-Xlint", "-Xfatal-warnings"),
-  scalaVersion := scala212,
+  scalaVersion := scala213,
```

Comment:
would be better to cross publish here instead of only using scala 2.13. can add something like this to `commonSettings` to cross publish.

```scala
lazy val commonSettings = Seq(
  scalaVersion := scala212Version,
  libraryDependencies += scalaTestArtifact,
  crossScalaVersions := Seq(
    scala212Version,
    scala213Version
  )... `rest of common settings`
)

```